plugin/al/usdtransaction/AL/usd/transaction/CMakeLists.txt

```diff
@@ -85,6 +85,25 @@ set_target_properties(${USDTRANSACTION_PYTHON_LIBRARY_NAME}
     LIBRARY_OUTPUT_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
 )
 
+# handle run-time search paths
+if(IS_MACOSX OR IS_LINUX)
+    mayaUsd_init_rpath(rpath ${USDTRANSACTION_PYTHON_LIBRARY_LOCATION})
+    mayaUsd_add_rpath(rpath "${CMAKE_INSTALL_PREFIX}/plugin/al/lib")
+    mayaUsd_install_rpath(rpath ${USDTRANSACTION_PYTHON_LIBRARY_NAME})
+endif()
+
+if(IS_WINDOWS)
+    set_target_properties(${USDTRANSACTION_PYTHON_LIBRARY_NAME}
+        PROPERTIES
+            SUFFIX ".pyd"
```

Comment:
When I was making my python debug changes I didn't realize this was a python library because it didn't have the .pyd extension. Good catch. But can you also make the change to add the _d to the name. Simply replace line 9 above with this:

if(IS_WINDOWS AND ${MAYAUSD_DEFINE_BOOST_DEBUG_PYTHON_FLAG})
    # On Windows when compiling with debug python the library must be named with _d.
    set(USDTRANSACTION_PYTHON_LIBRARY_NAME _${USDTRANSACTION_LIBRARY_NAME}_d)
else()
    set(USDTRANSACTION_PYTHON_LIBRARY_NAME _${USDTRANSACTION_LIBRARY_NAME})
endif()

Reply:
Roger that!